jwst/extract_1d/soss_extract/atoca.py

```diff
@@ -70,8 +74,13 @@ def __init__(self, wave_map, trace_profile, throughput, kernels,
             be used directly. N_ker is the length of the effective kernel
             and N_k_c is the length of the spectrum (f_k) convolved.
         global_mask : (N, M) array_like boolean, optional
-            Boolean mask of the detector pixels to mask for every extraction.
-        orders: list, optional
+            Boolean Mask of the detector pixels to mask for every extraction.
+            Should not be related to a specific order (if so, use `mask_trace_profile` instead).
+        mask_trace_profile: (N_ord, N, M) list or array of 2-D arrays[bool], optional
```

Comment:
minor nit: should have whitespace between end of argument name and ":" (applies to "orders" too)